src/main/java/de/tum/in/www1/artemis/service/connectors/localci/dto/LocalCIBuildJobQueueItem.java

```diff
@@ -8,7 +8,7 @@
 
 public record LocalCIBuildJobQueueItem(String id, String name, String buildAgentAddress, long participationId, String repositoryName, RepositoryType repositoryType,
         String commitHash, ZonedDateTime submissionDate, int retryCount, ZonedDateTime buildStartDate, ZonedDateTime buildCompletionDate, int priority, long courseId,
-        RepositoryType triggeredByPushTo, String dockerImage) implements Serializable {
+        RepositoryType triggeredByPushTo, String dockerImage, String result) implements Serializable {
```

Comment:
Why is "result" a string? For me it's not quite obvious what type of information is stored here.